.evergreen/config.yml

```diff
@@ -92,24 +105,28 @@ buildvariants:
   - build-and-test
   - build-and-test-asan
   - build-and-test-valgrind
+  - build-and-test-java
 - name: rhel76
   display_name: "RHEL 7.6"
   run_on: rhel76-test
   tasks:
   - build-and-test
   - build-and-test-asan
+  - build-and-test-java
 - name: macos
   display_name: "macOS 10.14"
   run_on: macos-1014
   tasks:
   - build-and-test
   - build-and-test-asan
+  - build-and-test-java
 - name: rhel72-zseries-test
   display_name: "RHEL 7.2 on zSeries"
   run_on: rhel72-zseries-test
   tasks:
   - build-and-test
   - build-and-test-asan
+  - build-and-test-java
 - name: windows-test
```

Comment:
Consider adding a TODO comment for windows (or commenting on ticket) so we remember to do that. Unless we're tracking that elsewhere.

Reply:
Opened https://jira.mongodb.org/browse/CDRIVER-3131